src/legends/searchable-discrete-color-legend.js

```diff
@@ -64,7 +67,7 @@ function SearchableDiscreteColorLegend(props) {
         <input
           type="search"
           placeholder={searchPlaceholder}
-          className="rv-search-wrapper__form__input"
+          className={inputClassName}
```

Comment:
i think better solution here would be to still keep that default class name available and always accept what the user gives, eg

```javascript
className={`rv-search-wrapper__form__input ${inputClassName}`}
```

and then the defaultProps would just be `inputClassName: ''`. I believe that this solution would be more in line with the way that we do class naming throughout the rest of the library.

Reply:
The issue with that approach is it still results in the styles from rv-search-wrapper__form__input being applied.  In my approach I am easily able to suppress that class and use my default input style.

An alternative would be to split the styles from rv-search-wrapper__form__input  into its own css file, making it easier to selectively include that styling.

Reply:
it seems like it will be pretty easy to override when there a multiple class names. to wit, if you had something like 
```scss
.my-custom-class.rv-search-wrapper__form__input {
//style style style
}
```
that would easily supercede the default styles. I'm guess I'm hesitant to start enabling the complete over-writing of styles? It makes sense here, but few other places.


An additional technique that we could employ here, (that we also employ other places) is offering a style prop at the top SearchableDiscreteColorLegend, that looks like

```
style: {
input: {},
colorLegend: {}
}
```

Reply:
With either of those techniques I'd be concerned about a new property I hadn't overriden on .rv-search-wrapper__form__input being added. Like if font-family the property was added to the css file in the future, then the search field would be using a different font from the rest of my app.

If css changes like that were considered breaking and only made on major releases, then I think your solution would work for me.